docs/application/dotnet/samples/ui/overview.md

```diff
@@ -174,10 +175,10 @@
 			</td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/ScriptLayoutSample" target="_blank"><strong>(T) Script Layout Sample (NUI)</strong></a></p>
-			<p>This sample application demonstrates how to use JSON file to create styles.<br />
-                        Applications often contain multiple controls that have an identical appearance.<br />
+			<p>This sample application demonstrates how to use JSON files to create styles.<br />
+            Application often contains multiple controls that have an identical appearance.<br />
 			Setting the appearance of each individual control can be repetitive and error prone.<br />
-			Instead, styles can be created that customize control appearance by grouping and settings properties available on the control type.
+			Therefore, you can create styles that customizes and control appearance by grouping and settings properties available on the control type.
```

Comment:
Isn't "setting properties" correct, instead of "settings properties"?

Reply:
Yes it is "setting properties".

Line 177: This sample application demonstrates how to use JSON files to create styles.
Application often contains multiple controls that have an identical appearance.
Setting the appearance of each individual control can be repetitive and error prone.
Therefore, you can create styles that customizes and controls appearance by grouping and setting properties available on the control type.

Reply:
done